Rakefile

```diff
@@ -2,18 +2,17 @@ require 'bundler/setup'
 require 'bundler/gem_tasks'
 require 'rspec/core/rake_task'
 
-require 'albacore'
```

Comment:
What did we use albacore for anyway?

Reply:
It's just a Ruby wrapper for building .Net projects. The same thing can be done via command line.